client/app/scripts/components/node-details/node-details-control-button.js

```diff
@@ -21,6 +22,7 @@ class NodeDetailsControlButton extends React.Component {
 
   handleClick(ev) {
     ev.preventDefault();
+    trackMixpanelEvent('scope.node.control.click', { id: this.props.control.id });
```

Comment:
I'd pass in `title: this.props.control.human` as well - maybe it would give us easier filtering in Mixpanel.

Reply:
Added title!